onnxruntime/core/providers/cpu/tensor/split.cc

```diff
@@ -16,7 +16,7 @@ namespace onnxruntime {
 namespace op_kernel_type_control {
 ORT_SPECIFY_OP_KERNEL_ARG_DEFAULT_TYPES_ALL_OPSETS(
     kCpuExecutionProvider, kOnnxDomain, Split, Input, 0,
-    float, int32_t, int64_t, uint8_t, std::string);
+    float, int8_t, int32_t, int64_t, uint8_t, std::string);
```

Comment:
@edgchen1 What's the best way to update whilst preserving hashes when using BuildKernelDefConstraintsFromTypeList?

Reply:
The current way looks good to me.

Reply:
docs/OperatorKernels.md needs to be updated as well. That would probably be worth adding it in kernel_def_hash_test.cc.